pom.xml

```diff
@@ -3275,6 +3275,18 @@
                         <!-- Fail tests longer than 20 minutes, prevent form random locking tests -->
                         <forkedProcessTimeoutInSeconds>1200</forkedProcessTimeoutInSeconds>
                         <excludedGroups>unstable</excludedGroups>
```

Comment:
Why exclude them?

Reply:
Sry you likely commented the wrong line so it is hard to understand (can u comment on the line again). The current line is already there before this PR :) that help separate the profile to run stable/unstable tests.